docs/database_api.md

```diff
@@ -88,44 +79,40 @@ this filename.
 }
 ```
 
-* F1 - F1 Score from model accuracy
-* accuracy - accuracy rate from model prediction
-* classificator - initials from used classificator
-* error - error rate from model prediction
-* fit_time - time from model fit using training dataset
+* `F1` - F1 Score from model accuracy
+* `accuracy` - Accuracy rate from model prediction
+* `classificator` - Initials from used classificator
+* `error` - Error rate from the model prediction
```

Comment:
Please replace `error` field by `filename` field

Reply:
You can copy the `filename` definition from any other JSON microservice.

Reply:
Alright, I was wondering why I couldn't see error in the JSON